pandas-stubs/core/strings.pyi

```diff
@@ -87,19 +87,19 @@ class StringMethods(NoNewAttributesMixin, Generic[T]):
     def normalize(self, form) -> T: ...
     def index(self, sub, start: int = ..., end=...) -> T: ...
     def rindex(self, sub, start: int = ..., end=...) -> T: ...
-    len = ...
-    lower = ...
-    upper = ...
-    title = ...
-    capitalize = ...
-    swapcase = ...
-    casefold = ...
-    isalnum = ...
-    isalpha = ...
-    isdigit = ...
-    isspace = ...
-    islower = ...
-    isupper = ...
-    istitle = ...
-    isnumeric = ...
-    isdecimal = ...
+    def len(self) -> Series[float]: ...
```

Comment:
It is unfortunately `float` because when there is ` NaN` you get values like `3.0, 4.0, NaN` .  Plus int is a float.

Reply:
The docstring is funny

https://pandas.pydata.org/docs/reference/api/pandas.Series.str.len.html

because it says int but the example shows float.

Reply:
> The docstring is funny
> 
> https://pandas.pydata.org/docs/reference/api/pandas.Series.str.len.html
> 
> because it says int but the example shows float.

Should probably create an issue for that over in pandas.